treeherder/config/settings.py

```diff
@@ -132,13 +132,19 @@
 #   'mysql://username:password@host:optional_port/database_name'
 #
 # which django-environ converts into the Django DB settings dict format.
-LOCALHOST_MYSQL_HOST = 'mysql://root@{}:3306/treeherder'.format(
+MYSQL_LOCALHOST = 'mysql://root@{}:3306/treeherder'.format(
     'localhost' if IS_WINDOWS else '127.0.0.1'
 )
+DATABASE_URL = env.db_url('DATABASE_URL', default=MYSQL_LOCALHOST)
 DATABASES = {
-    'default': env.db_url('DATABASE_URL', default=LOCALHOST_MYSQL_HOST),
+    'default': DATABASE_URL,
 }
 
+SKIP_INGESTION = False
+# Skip data ingestion if it uses something different than localhost
+if DATABASE_URL != MYSQL_LOCALHOST:
+    SKIP_INGESTION = True
```

Comment:
@imskr I've commited some changes since they were my preference on how it should be written.
Have a look to see how I made it easier to read.

Reply:
Cool. Looks clean! Thanks